WooCommerce/Classes/Authentication/AuthenticationManager.swift

```diff
@@ -75,7 +77,12 @@ class AuthenticationManager: Authentication {
                                                                 isWPComLoginRequiredForSiteCredentialsLogin: true,
                                                                 isWPComMagicLinkPreferredToPassword: isWPComMagicLinkPreferredToPassword,
                                                                 isWPComMagicLinkShownAsSecondaryActionOnPasswordScreen:
-                                                                    isWPComMagicLinkShownAsSecondaryActionOnPasswordScreen)
+                                                                    isWPComMagicLinkShownAsSecondaryActionOnPasswordScreen,
+                                                                enableWPComLoginOnlyInPrologue: isSimplifiedLoginI1Enabled,
+                                                                enableSiteCreation: isStoreCreationMVPEnabled,
+                                                                enableSocialLogin: !isSimplifiedLoginI1Enabled,
+                                                                emphasizeEmailForWPComPassword: isSimplifiedLoginI1Enabled,
+                                                                wpcomPasswordInstructions: AuthenticationConstants.wpcomPasswordInstructions)
```

Comment:
do we want to pass the string only under `isSimplifiedLoginI1Enabled`?

Reply:
Good catch! Updated in fdebdc3e95e637d7299d56f3b60d3040b22d8f0c.